From fd4b18787a6415c4a03058911a3e0fc8080d0f19 Mon Sep 17 00:00:00 2001
From: Manuel Naranjo <mnaranjo@irobot.com>
Date: Fri, 5 Aug 2016 12:14:50 -0300
Subject: [PATCH] Don't install the libraries in win32

Prevent installing libraries in win32 so those can be built with
makepkg-mingw and packaged for msys2
---
 qtpropertybrowser/buildlib/buildlib.pro   | 4 ----
 qtservice/buildlib/buildlib.pro           | 4 ----
 qtsingleapplication/buildlib/buildlib.pro | 4 ----
 3 files changed, 12 deletions(-)

diff --git a/qtpropertybrowser/buildlib/buildlib.pro b/qtpropertybrowser/buildlib/buildlib.pro
index 8da3de2..7c7d607 100644
--- a/qtpropertybrowser/buildlib/buildlib.pro
+++ b/qtpropertybrowser/buildlib/buildlib.pro
@@ -5,9 +5,5 @@ win32|mac:!wince*:!win32-msvc:!macx-xcode:CONFIG += debug_and_release build_all
 include(../src/qtpropertybrowser.pri)
 TARGET = $$QTPROPERTYBROWSER_LIBNAME
 DESTDIR = $$QTPROPERTYBROWSER_LIBDIR
-win32 {
-    DLLDESTDIR = $$[QT_INSTALL_BINS]
-    QMAKE_DISTCLEAN += $$[QT_INSTALL_BINS]\\$${QTPROPERTYBROWSER_LIBNAME}.dll
-}
 target.path = $$DESTDIR
 INSTALLS += target
diff --git a/qtservice/buildlib/buildlib.pro b/qtservice/buildlib/buildlib.pro
index 1e51cc1..00ac78c 100644
--- a/qtservice/buildlib/buildlib.pro
+++ b/qtservice/buildlib/buildlib.pro
@@ -5,9 +5,5 @@ win32|mac:!wince*:!win32-msvc:!macx-xcode:CONFIG += debug_and_release build_all
 include(../src/qtservice.pri)
 TARGET = $$QTSERVICE_LIBNAME
 DESTDIR = $$QTSERVICE_LIBDIR
-win32 {
-    DLLDESTDIR = $$[QT_INSTALL_BINS]
-    QMAKE_DISTCLEAN += $$[QT_INSTALL_BINS]\\$${QTSERVICE_LIBNAME}.dll
-}
 target.path = $$DESTDIR
 INSTALLS += target
diff --git a/qtsingleapplication/buildlib/buildlib.pro b/qtsingleapplication/buildlib/buildlib.pro
index 37dddcd..3e7098f 100644
--- a/qtsingleapplication/buildlib/buildlib.pro
+++ b/qtsingleapplication/buildlib/buildlib.pro
@@ -5,9 +5,5 @@ win32|mac:!wince*:!win32-msvc:!macx-xcode:CONFIG += debug_and_release build_all
 include(../src/qtsingleapplication.pri)
 TARGET = $$QTSINGLEAPPLICATION_LIBNAME
 DESTDIR = $$QTSINGLEAPPLICATION_LIBDIR
-win32 {
-    DLLDESTDIR = $$[QT_INSTALL_BINS]
-    QMAKE_DISTCLEAN += $$[QT_INSTALL_BINS]\\$${QTSINGLEAPPLICATION_LIBNAME}.dll
-}
 target.path = $$DESTDIR
 INSTALLS += target
-- 
2.7.2

